solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -550,14 +560,8 @@ contract RandomBeacon is Ownable {
 
         groups.activateCandidateGroup();
         dkg.complete();
-
-        // TODO: Should result submitter receive reward if they failed to call
-        //       this function?
-        // TODO: Ensure this function is as cheap as possible and it is
-        //       profitable for the DKG result submitter to call it. Consider
-        //       giving the submitter some time to approve the result and if
-        //       don't, pay the submitter's reward to anybody who calls this
-        //       function.
+        // TODO: Check if this function is cheap enough and it will be
+        //       profitable for the DKG result submitter to call it.
```

Comment:
What is the gas cost of `submitDkgResult` and `approveDkgResult`? Maybe it is something we should no longer be concerned about and the only challenge is to set `dkgResultSubmissionReward` high enough.

Reply:
Awesome. Then I do not think we need to worry about it. We can keep that TODO and double-check once we finish all the other work, no need to block this PR on it.